### ISSUE TITLE
chrome version  111.0.5563.65  no work

### ISSUE THREAD

**Issue by sxin-h** [webgpu/webgpu-samples#236]
RT

**Comment by kainino0x** [comment]
Confirmed not working for me on Mac M1.

```
Google Chrome	111.0.5563.64 (Official Build) (arm64) 
Revision	c710e93d5b63b7095afe8c2c17df34408078439d-refs/branch-heads/5563@{#995}
OS	macOS Version 13.2.1 (Build 22D68)
```

**Comment by PollRobots** [comment]
Confirmed not working on Windows 11

Version 111.0.5563.65 (Official Build) (64-bit)

If I run 
``` JavaScript
navigator.gpu.requestAdapter().then(a => a.requestAdapterInfo()).then(console.log);
``` 

This prints out a correct looking GPUAdapterInfo

If I run

``` JavaScript
navigator.gpu.requestAdapter().then(a => a.requestDevice()).then(console.log).catch(console.error);
```

Then it returns the promise (as expected), but never logs to the console, the promise never appears to resolve.

**Comment by austinEng** [comment]
This was indeed a Chrome bug which has been fixed already in other branches, but missed in 111. It has already been patched and will be fixed in the next update to the 111 branch.

**Comment by kainino0x** [comment]
Fixed in 111.0.5563.115:
https://chromiumdash.appspot.com/commit/09e2ab9d925d4462bf04d71d5809caf9f585eabe
